### ISSUE TITLE
New design for static site navbar

### ISSUE THREAD

**Issue by logan-r** [hackmcgill/mchacks7#89]
Navbar on static site should now look like so: (minus the apply link when applications aren't open)

![image](https://user-images.githubusercontent.com/3229837/97786227-a8ff4e80-1b80-11eb-9052-c6a373b1ec75.png)

**Comment by pierreTklein** [comment]
What could be really cool: the static site can make a GET call to the API for the hackathon settings, and then figure out if applications are open.

**Comment by logan-r** [comment]
Yeah I agree! We could also use same logic to switch between the sign up for mailing list textbox and apply now button
